frontend/src/scenes/insights/InsightNav/insightNavLogic.ts

```diff
@@ -31,6 +32,8 @@ export const insightNavLogic = kea<insightNavLogicType>([
             ['featureFlags'],
             insightDataLogic(props),
             ['query'],
+            userLogic,
+            ['user'],
```

Comment:
What does this do?

Reply:
That is very important structural code (also, it isn't necessary any more)